test/Motor.Extensions.Hosting_UnitTest/MultiOutputServiceAdapterTests.cs

```diff
@@ -162,10 +161,15 @@ public async Task HandleMessageAsync_ConverterReturnsMultipleResults_PublisherIs
             ;
         }
 
-        private Task<IEnumerable<MotorCloudEvent<string>>> CreateReturnValues(params string[] data)
+#pragma warning disable 1998
```

Comment:
Why is the warning disabled for this function?

Reply:
It is much easier to declare a function async and ignore the warning instead of doing it non async.

Reply:
Ok, but why is there a warning at all? I guess it has a reason to warn about the way async is used here.

Reply:
There is no await in the function.

Reply:
Wouldn't is be cleaner to actually run async then?
E.g.:
```csharp
yield return wait Task.Run(() => MotorCloudEvent.CreateTestCloudEvent(t, new Uri("test://non")));
```
In that case there is no warning and the function is async as it states.

Reply:
To clarify: This is only a nitpick for me and I would not stop the merge because of the disabled warning. I was just curious why it is handled that way.

Reply:
You could write it as following:
```
yield return await Task.FromResult(MotorCloudEvent.CreateTestCloudEvent(t, new Uri("test://non"))));
```
In that case, the function has also no async warning, but don' t require to build up internal an extra thread.

Reply:
Even better idea. I think this is much cleaner than disabling the warning.